basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/jmsbridge/JMSBridgeAttributesTestCase.java

```diff
@@ -153,7 +154,7 @@ public void toggleAddMessageIDInHeader() throws Exception {
     public void editClientID() throws Exception {
         final String VALUE = RandomStringUtils.randomAlphanumeric(5);
 
-        ModelNodeResult defaultValueResult = operations.readAttribute(JMS_BRIDGE_ADDRESS, CLIENT_ID_IDENTIFIER);
+        ModelNodeResult defaultValueResult = operations.readAttribute(JMS_BRIDGE_ADDRESS, CLIENT_ID_IDENTIFIER, ReadAttributeOption.NOT_INCLUDE_DEFAULTS);
```

Comment:
Renaming it to `origValue` would make more sense as you've changed it from default value to actually the original value, which might be also `undefined` value instead of the default value.